examples/wfs.js

```diff
@@ -115,7 +115,7 @@ scaler = {
     },
 };
 
-view.addFrameRequester(scaler);
+view.addFrameRequester(itowns.MAIN_LOOP_EVENTS.BEFORE_RENDER, scaler.update);
```

Comment:
(not very important, but as you'll amend this PR anyway, scaler could be a simple function)

Reply:
Good call, done !